testdata/picard/quality/insert_size_metrics_test_fake.quality_yield_metrics

```diff
@@ -0,0 +1,10 @@
+## htsjdk.samtools.metrics.StringHeader
+# CollectQualityYieldMetrics INPUT=/Users/farjoun/Picard-public/testdata/picard/sam/insert_size_metrics_test.sam OUTPUT=/var/folders/cj/97bf7h5j7l194mqhrdg88rdm0000gp/T/test16801848535468386074.quality_yield_metrics    USE_ORIGINAL_QUALITIES=true INCLUDE_SECONDARY_ALIGNMENTS=false INCLUDE_SUPPLEMENTAL_ALIGNMENTS=false ASSUME_SORTED=true STOP_AFTER=0 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
+## htsjdk.samtools.metrics.StringHeader
+# Started on: Tue Sep 07 15:33:31 EDT 2021
+
+## METRICS CLASS	picard.analysis.CollectQualityYieldMetrics$QualityYieldMetrics
+TOTAL_READS	PF_READS	READ_LENGTH	TOTAL_BASES	PF_BASES	Q20_BASES	PF_Q20_BASES	Q30_BASES	PF_Q30_BASES	Q20_EQUIVALENT_YIELD	PF_Q20_EQUIVALENT_YIELD
+50	50	100	5250	5250	3530	3530	3140	3140	6490	6490
```

Comment:
Shouldn't the READ_LENGTH here be 105? Or is that why this file is "fake"?

Reply:
no...that's a typo on my part. since that field gets recalculated from the number of bases & number of reads, it doesn't matter...but I'll fix it.

Reply:
Thanks, I think it's clearer even though that number doesn't actually get used.